modules/data.atmosphere/R/met.process.R

```diff
@@ -133,11 +133,12 @@ met.process <- function(site, input_met, start_date, end_date, model,
   
   
   # setup site database number, lat, lon and name and copy for format.vars if new input
-  latlon <- PEcAn.DB::query.site(site$id, con = con)[c("lat", "lon")] 
-  new.site <- data.frame(id = as.numeric(site$id), 
-                         lat = latlon$lat, 
-                         lon = latlon$lon)
-  str_ns <- paste0(new.site$id %/% 1e+09, "-", new.site$id %% 1e+09)
+  latlon <- NULL
```

Comment:
Why is latlon defined to be null here? Why not just delete this line and the `, latlon = latlon` in the function call as you already set the default for latlon to NULL.